srcpkgs/voidvault/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'voidvault'
+pkgname=voidvault
+version=1.3.1
+revision=1
+only_for_archs="i686 x86_64"
```

Comment:
Why?

Reply:
Musl has different system directory layout. It could be made to work with musl I suppose... but for now it is only working on glibc systems.

Reply:
This is enough to keep it out of the repositories.  Void supports musl as a first class system, and any installer that doesn't also support musl with full feature parity cannot be accepted to the repo.

Reply:
Agreed with the above. A void installed must work on any arch that might care to use an installer (all except Single-Board-Computer-Only archs)

Reply:
I still hate seeing `only_for_archs`.

Reply:
Would you rather I put `noarch=yes`, then?